### PR TITLE
Make core probes configurable

### PR DESCRIPTION
**Pull request by bpehling**
Make core probes configurable. On quota migration the core pods require more time with large amounts of data and should be configurable to allow migration to complete.

Issue: On quota migration from 1.7 to 1.9 the pods were restarting since the checks weren't ready and restarted the containers which resulted in restarting the migration before completion.

Signed-off-by: Ben Pehling <shinjipehling@gmail.com>

### CODE REVIEW

**Comment by ywk253100** [comment]
The PR is fixing #371, we'll estimate the solution provided by the PR. Thanks @bpehling

File: values.yaml [hunk 1]
```diff
@@ -300,6 +300,12 @@ core:
     repository: goharbor/harbor-core
     tag: dev
   replicas: 1
+  livenessProbe:
```
Comment (ywk253100) [inline values.yaml]: @bpehling I think we can just make the `initialDelaySeconds` of `livenessProbe` configurable to fix the upgrading issue and with less configuration items for users.

Please add comment for the item to explain what is it for and document it in the `README.md`

Comment (ywk253100) [reply values.yaml]: And larger the delay time, maybe 300 is a reasonable value?

Comment (bpehling) [reply values.yaml]: I think 300 is reasonable, fixed

Comment (ywk253100) [reply values.yaml]: Thanks

**Comment by ywk253100** [comment]
@bpehling Could you cherry pick the commit into the `1.2.0` branch? We'll release 1.2.1 to include some fixes.

**Pull request merged**